2-projects/1-todo-list/script.js

```diff
@@ -12,14 +15,68 @@ let todos = [
 
 populateTodoList(todos);
 
+function createTodoElem (taskText, parent){
+  let li = document.createElement('li');
+  li.innerText = taskText;
+  li.classList.add("list-group-item", "d-flex", "justify-content-between", "align-items-center");
+
+  //create icons to remove and delete list items and add it to li
+  let container = document.createElement('span');
```

Comment:
Could use more descriptive method names so that it is much more clearer on what the variable is